data/glossary.csv

```diff
@@ -359,7 +358,7 @@ Les services internes sont uniquement des activités et des ressources offerts a
 * les transitions gouvernementales
 "
 "MAINS","Main Estimates","Budget principal des dépenses","The Main Estimates is a document that directly supports the appropriation act for the beginning of a fiscal year. It provides a listing of the resources required by individual departments and agencies for the upcoming fiscal year in order to deliver the programs for which they are responsible. It identifies the spending authorities (votes) and the amounts to be included in subsequent appropriation bills that Parliament will be asked to approve to enable the government to proceed with its spending plans. It is traditionally known as the “Blue Book”.","Le budget principal des dépenses est le document qui appuie directement la loi de crédits adoptée pour commencer un exercice. On y trouve une liste des ressources dont les ministères et organismes auront besoin pour l’exécution des programmes dont ils sont responsables au cours du prochain exercice. Il fait état des autorisations de dépenser (crédits) et des montants qui figureront dans les prochains projets de loi de crédits que le gouvernement cherchera à faire approuver par le Parlement pour mettre en œuvre ses plans de dépenses. Ce document est traditionnellement appelé « Livre bleu »."
-"MAN_IGOC","Description / Raison d’être","Description / raison d’être","The purpose or reason for which an organization exists.","Fondement de l’existence d’une organisation."
+"MAN_IGOC","Description","Raison d’être","The purpose or reason for which an organization exists.","Fondement de l’existence d’une organisation."
```

Comment:
I think this one's supposed to show bilingual, at least in English, since "Raison d’être" is sometimes borrowed in English.

Reply:
ah okay I'll revert this one then